src/main/java/org/breedinginsight/model/delta/DeltaEntity.java

```diff
@@ -0,0 +1,43 @@
+package org.breedinginsight.model.delta;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
+import java.util.UUID;
+
+public abstract class DeltaEntity<T> {
+
+    protected final Gson gson;
+
+    @NonNull
+    protected final T brAPIObject;
+
+    @Setter
+    @Getter
+    protected UUID id;
```

Comment:
This field needs to be deleted. Deltabreed ids are stored as xrefs in the brapi objects and should have a getter and setter that fetches and writes that xref to the brapi object.